go.mod

```diff
@@ -23,11 +23,11 @@ require (
 	github.com/hashicorp/golang-lru v0.5.5-0.20210104140557-80c98217689d
 	github.com/ipfs/go-cid v0.3.2
 	github.com/kardianos/service v1.2.0
-	github.com/libp2p/go-libp2p v0.24.2
+	github.com/libp2p/go-libp2p v0.24.3-0.20230207035812-313b080ea4e2
```

Comment:
this will have to be updated to the release version once it's out.

Reply:
would it make sense to have this comment on this line as well? just to serve as reminder to developers in the future, since this comment will fade once this PR is closed